app/src/marketplace/containers/ProductPage/DataUnionStats.jsx

```diff
@@ -92,11 +95,20 @@ const UnstyledDataUnionStats = ({
                                 />
                             </TimeSeriesGraph.Header>
                             <TimeSeriesGraph.Body>
-                                <MembersGraph
-                                    joinPartStreamId={joinPartStreamId}
-                                    memberCount={memberCount.total}
-                                    shownDays={days}
-                                />
+                                {dataUnion && dataUnion.version && dataUnion.version === 2 && (
+                                    <MembersGraphV2
+                                        memberCount={memberCount.total}
+                                        shownDays={days}
+                                        dataUnionAddress={dataUnion.id}
+                                    />
+                                )}
+                                {joinPartStreamId && (
```

Comment:
Compare version instead of joinpart stream for clarity?

Reply:
Yeah, I'll change that!